### ISSUE TITLE
Keyboard scrolling in scrollOverflow section works after click only and overflow reset not working

### ISSUE THREAD

**Issue by davITnemec** [alvarotrigo/fullPage.js#4470]
### Descriptions
Hello, I have two issues with scrollOverflow sections: 

1. I have a section with scrollOverflow, the keyboard navigation doesn't work. I need to first click inside the section and then I can use the arrows, PageUp, PageDown etc. It's happening in every browser and only in scrollOverflow sections. If the section is not scrollOverflow, the keyboard navigation works perfectly and automatically.
2. When I set up in config `scrollOverflowReset: true,` it doesn't have any effect. The scrollOverflow section is still in the end. What I'm doing wrong?

### Steps to reproduce it
1. Open the website - https://martincova-art.cz/
2. Insert password - alvarotrigoIStheBEST
3. Test it - the second section

### Versions 
Latest versions (Chrome, FullPage.js, Divi)

**Comment by alvarotrigo** [comment]
Try using the dev version 4.0.12:
https://github.com/alvarotrigo/fullPage.js/tree/dev/dist

The keyword controls seem to be working fine on this reproduction:
https://codepen.io/alvarotrigo/pen/QWrzeVv

If you still think there's an issue make sure to update the codepen to create an isolated reproduction of your issue.

**Comment by davITnemec** [comment]
@alvarotrigo here is the jsfiddle - https://jsfiddle.net/alvarotrigo/ea17skjr/ and also the video with description what I'm dealing with - https://ctrlv.tv/oCmb

**Comment by alvarotrigo** [comment]
@davITnemec the option `scrollOverflowReset` requires a paid extensions as detailed on the <a href="https://alvarotrigo.com/fullPage/docs" target="_blank">fullpage.js documentation</a>.

Regarding the keyboard navigation, I'm not able to reproduce the issue on my side.
Can you reproduce your issue here?
https://jsfiddle.net/bjd2hf3n/

And if so, can you specify the following?
- Web browser (and version)
- Operative system
